packages/web-app-files/src/components/FilesList/ResourceTable.vue

```diff
@@ -112,6 +114,21 @@
         + {{ item.tags.length - 2 }}
       </oc-tag>
     </template>
+    <template #manager="{ item }">
+      <slot name="manager" :resource="item" />
+    </template>
+    <template #members="{ item }">
+      <slot name="members" :resource="item" />
+    </template>
+    <template #totalQuota="{ item }">
+      <slot name="totalQuota" :resource="item" />
+    </template>
+    <template #usedQuota="{ item }">
+      <slot name="usedQuota" :resource="item" />
+    </template>
+    <template #remainingQuota="{ item }">
+      <slot name="remainingQuota" :resource="item" />
+    </template>
```

Comment:
Hmmm does it really make sense to add all these columns to the generic resource table? 
Wouldn't it make more sense to add these dynamically from the parent component where needed?

Reply:
but we need them as fields, or what do you mean by dynamically from the parent?